windows/internal/build_magma.bat

```diff
@@ -0,0 +1,92 @@
+@setlocal
+
+set MAGMA_VERSION=2.5.4
+
+set CUVER_NODOT=%CUDA_VERSION%
+set CUVER=%CUVER_NODOT:~0,-1%.%CUVER_NODOT:~-1,1%
+
+set CONFIG_LOWERCASE=%CONFIG:D=d%
+set CONFIG_LOWERCASE=%CONFIG_LOWERCASE:R=r%
+set CONFIG_LOWERCASE=%CONFIG_LOWERCASE:M=m%
+
+echo Building for configuration: %CONFIG_LOWERCASE%, %CUVER%
+
+:: Download Ninja
+curl -k https://s3.amazonaws.com/ossci-windows/ninja_1.8.2.exe --output C:\Tools\ninja.exe
+
+dir C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC
+set "PATH=C:\Tools;C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v%CUVER%\bin;C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v%CUVER%\libnvvp;%PATH%"
+set CUDA_PATH=C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v%CUVER%
+set NVTOOLSEXT_PATH=C:\Program Files\NVIDIA Corporation\NvToolsExt
+
+mkdir magma_cuda%CUVER_NODOT%
+cd magma_cuda%CUVER_NODOT%
+
+:: First install MKL, which provides BLAS and LAPACK API
+:: Download and install from:
+:: curl https://s3.amazonaws.com/ossci-windows/w_mkl_2018.2.185.exe -k -O
+:: .\w_mkl_2018.2.185.exe
+:: Follow the installer steps and install MKL to default path
+
+pushd %CD%
+call "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Auxiliary\Build\vcvarsall.bat" x86_amd64
```

Comment:
Since you enabled `ilammy/msvc-dev-cmd@dd5e2fa0a7de1e7929605d9ecc020e749d9856a3` in GHA, would you please remove this line?